content/docs/installation.md

```diff
@@ -4,15 +4,129 @@ description: Get up and running on Linux and a variety of container platforms
+
+The KRE also allows you to deploy security playbooks (via serverless functions) that can take action when Falco rules are violated. Refer to the [Response Engine documentation](https://github.com/falcosecurity/kubernetes-response-engine/tree/master/playbooks) on how to deploy the included playbooks.
+
+### DaemonSet Manifests
+To run Falco as a Kubernetes [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/), follow the instructions below. These are the `generic` instructions for Kubernetes. See below for Kubernetes platform specific instructions.
+
+1. Clone the [Falco repo](https://github.com/falcosecurity/falco/) and change to the directory with the manifests.
+```shell
+git clone https://github.com/falcosecurity/falco/
+cd falco/integrations/k8s-using-daemonset
+```
+2. Create a service account and RBAC permissions which Falco will run as. This will allow Falco to connect to the Kubernetes API server to fetch resource metadata. 
+```shell
+kubectl apply -f k8s-with-rbac/falco-account.yaml
+```
+3. Create a service for the Falco pods. This will allow Falco to recieve [Kubernetes Audit Log Events](event-sources/kubernetes-audit). If you're not planning on using this feature, you can skip this step.
+```shell
+kubectl apply -f k8s-with-rbac/falco-service.yaml
+```
+
+4. The Daemon Set also relies on a Kubernetes ConfigMap to store the Falco configuration and make the configuration available to the Falco Pods. This allows you to manage custom configuration without rebuilding and redeploying the underlying Pods. In order to create the ConfigMap you'll need to first need to copy the required configuration from their location in this GitHub repo to the `k8s-with-rbac/falco-config/` directory (please note that you will need to create the k8s-with-rbac/falco-config directory). Any modification of the configuration should be performed on these copies rather than the original files.
+
+```shell
+mkdir -p k8s-with-rbac/falco-config
+k8s-using-daemonset$ cp ../../falco.yaml k8s-with-rbac/falco-config/
+k8s-using-daemonset$ cp ../../rules/falco_rules.* k8s-with-rbac/falco-config/
+k8s-using-daemonset$ cp ../../rules/k8s_audit_rules.yaml k8s-with-rbac/falco-config/
+```
+
+5. Any custom rules for your environment can be added to into the `falco_rules.local.yaml` file and they will be picked up by Falco at start time. You can also modify the `falco.yaml` file to change any [configuration options](configuration/) required for your deployment. Create the configmap with the below command.
+```shell
+kubectl create configmap falco-config --from-file=k8s-with-rbac/falco-config
+```
+
+6. With the dependencies of the configmap created, the daemonset can now be created.
```

Comment:
With the dependencies of the configMap created, you can now create the DaemonSet.